### PR TITLE
chore: correct name for calling RelationshipIngresses

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Update variable name for caller for RelationshipIngresses to make reading clearer

## References
Close https://github.com/openfga/openfga/issues/612

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: pkg/server/commands/connected_objects.go [hunk 1]
```diff
@@ -169,7 +169,7 @@ func (c *ConnectedObjectsCommand) streamedConnectedObjects(
 			r := &reverseExpandRequest{
 				storeID:          storeID,
 				ingress:          innerLoopIngress,
-				sourceObjectRef:  sourceObjRef,
+				sourceObjectRef:  targetObjRef,
```
Comment (miparnisari) [inline pkg/server/commands/connected_objects.go:172]: this is confusing

Comment (adriantam) [reply pkg/server/commands/connected_objects.go]: updated

**Review by miparnisari — DISMISSED** [review]

**Comment by miparnisari** [comment]
https://github.com/openfga/openfga/blob/1f63f98fc1a30e752a1b6d6d38d1ef2041f79875/internal/graph/graph.go#L134-L139

do you mind renaming `rewrite` to `targetRewrite` as well? and also in the callers

File: internal/graph/graph.go [hunk 2]
```diff
@@ -258,6 +258,6 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 	case *openfgapb.Userset_Intersection, *openfgapb.Userset_Difference:
 		return nil, ErrNotImplemented
 	default:
-		panic("unexpected userset rewrite encountered")
+		panic("unexpected userset targetRewrite encountered")
```
Comment (miparnisari) [inline internal/graph/graph.go:261]: undo

Comment (adriantam) [reply internal/graph/graph.go]: done

File: internal/graph/graph.go [hunk 3]
```diff
@@ -247,8 +247,8 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 	case *openfgapb.Userset_Union: // e.g. target = define viewer as self or writer
 		var res []*RelationshipIngress
 		for _, child := range t.Union.GetChild() {
-			// we recurse through each child rewrite
-			childResults, err := g.findIngressesWithRewrite(target, source, child, visited)
+			// we recurse through each child targetRewrite
```
[the remark this answers is not in the document]
Comment (adriantam) [reply internal/graph/graph.go]: done

**Review by miparnisari — APPROVED** [review]

**Review by jon-whit — APPROVED** [review]

**Pull request merged**